packages/contracts-bedrock/src/L1/L1StandardBridge.sol

```diff
@@ -75,31 +81,38 @@ contract L1StandardBridge is StandardBridge, ISemver {
     );
 
     /// @notice Semantic version.
-    /// @custom:semver 2.2.1-beta.1
-    string public constant version = "2.2.1-beta.1";
+    /// @custom:semver 2.3.1-beta.1
+    // TODO: check version set
+    string public constant version = "2.3.1-beta.1";
 
     /// @notice Address of the SuperchainConfig contract.
     ISuperchainConfig public superchainConfig;
 
     /// @notice Address of the SystemConfig contract.
     ISystemConfig public systemConfig;
 
+    address public balanceClaimer;
```

Comment:
Missin natspedc

Reply:
also storing it casted to `IBalanceClaimer` for neatness :)

Reply:
+1 to IBalanceWithdrawer